tests/storage/datavolume.go

```diff
@@ -1340,3 +1373,18 @@ func renderVMWithRegistryImportDataVolume(containerDisk cd.ContainerDisk, storag
 	)
 	return libstorage.RenderVMWithDataVolumeTemplate(dv)
 }
+
+func renderVMWithRegistryImportDataVolumePVCAPI(containerDisk cd.ContainerDisk, storageClass string) *v1.VirtualMachine {
```

Comment:
`renderVMWithRegistryImportDataVolumePVC` or better yet just in-line it if it's only called once for now.